### DISCUSSION TITLE
SDK Installation

### DISCUSSION THREAD

**Discussion by AlexAS81** [OP-TED/eForms-SDK#146] [Q&A] [answered]
Hi all,

Could someone explain the steps to follow to install the SDK in a Java project and how to generate the eForms with the SDK?

Thank a lot
Alex

**Comment by rouschr** [comment, marked as answer]
Hello,

for the installation in a Java project you need to use Maven, the SDK has a pom.xml.

For the eForms there is a discussion happening here that could help you:
https://github.com/OP-TED/eForms-SDK/discussions/126

Also see about the Editor demo project:
https://github.com/OP-TED/eforms-notice-editor
